test/Resolver/DependencyResolverTest.php

```diff
@@ -132,7 +135,6 @@ private function mockDefinition(array $definition)
             $mock->method('getClassDefinition')
                 ->with($class)
                 ->willReturn($this->mockClassDefinition(
-                    $class,
```

Comment:
Why do you remove this parameter?

Reply:
https://github.com/laminas/laminas-di/blob/3.2.x/test/Resolver/DependencyResolverTest.php#L78
Ist not used inside mockClassDefinition. 
But as you said only code style changes, no refactorings. ;-). Im going to revert those last changes and I should be good.